### DISCUSSION TITLE
Move the cursor >1 lines

### DISCUSSION THREAD

**Discussion by victor141516** [zed-industries/zed#6637] [Feature Requests]
Yesterday I found this nice project and I have most of my VSCode workflows adapted. I just miss one thing I use in VSCode:
I have this config in VSCode:
```json
{
	"key": "cmd+up",
	"command": "cursorMove",
	"args": {
		"by": "line",
		"to": "up",
		"value": 10
	},
	"when": "textInputFocus"
},
{
	"key": "cmd+down",
	"command": "cursorMove",
	"args": {
		"by": "line",
		"to": "down",
		"value": 10
	},
	"when": "textInputFocus"
}
```

It will move the cursor by 10 lines with cmd+up/down

Trying to replicate that on Zed  I started from this:
```json
{
  "context": "Editor",
  "bindings": {
    "cmd+up": "editor::MoveDown"
  }
}
```

And I tried to change the action to:
```json
["editor::MoveDown", 10]
```
or
```json
["editor::MoveDown", { "lines": 10 }]
```

But none of those worked. I could not find any documentation about those actions and the available options for them.

Is this something we can currently do somehow? or is this not yet implemented?

Thank you!

**Comment by aurexav** [comment]
```
"ctrl-up": ["editor::MoveUpByLines", { "lines": 15 }],
"ctrl-down": ["editor::MoveDownByLines", { "lines": 15 }],
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
